.github/workflows/release.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: Release
+
+"on":
+  workflow_run:
+    workflows: [Checks]
+    types: [completed]
+    branches: [main]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  info:
+    name: Info
+    runs-on: ubuntu-latest
+    outputs:
+      head_sha: ${{ steps.head_sha.outputs.head_sha }}
+      timestamp: ${{ steps.timestamp.outputs.timestamp }}
+
+    steps:
+      - name: Git Info
+        id: head_sha
+        env:
+          GH_TOKEN: ${{ github.token }}
+          GH_COBRA: 1
+        run: |
+          echo head_sha=$(gh api /repos/zregvart/enterprise-contract-controller/git/matching-refs/heads/main --jq '.[0].object.sha') >> "$GITHUB_OUTPUT"
```

Comment:
Is this right? I see a personal repo in the query. 

I'm also not sure why we want to echo `head_sha=<the-sha>`. Easier to debug? Why not just use the `steps.head_sha.outputs.head_sha`?

Reply:
> Is this right? I see a personal repo in the query.

Thanks, this was mistakenly left from trying this out on my fork.

> I'm also not sure why we want to echo `head_sha=<the-sha>`. Easier to debug? Why not just use the `steps.head_sha.outputs.head_sha`?

See https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
Thanks! Forgot about that.